### ISSUE TITLE
Update service operation acceptance tests to use new sccli service

### ISSUE THREAD

**Issue by brgnepal** [projectatomic/vagrant-service-manager#280]
Currently, in the implementation of acceptance test for service operation, [`systemctl`](https://github.com/projectatomic/vagrant-service-manager/blob/master/features/support/env.rb#L96) is used. With the latest `sccli`, it is now easy to manage these service specific operations as per [`adb-utils README](https://github.com/projectatomic/adb-utils#adb-utils)

```
sccli [-h] {kubernetes,openshift,docker} ...

CLI for managing services in ADB/CDK

optional arguments:
-h, --help            show this help message and exit

subcommands:
Manage services for openshift|docker|kubernetes

{kubernetes,openshift,docker}
kubernetes          start|restart|status|stop (default:start)
openshift           start|restart|status|stop (default:start)
docker              start|restart|status|stop (default:start)
```

**Comment by brgnepal** [comment]
Blocked on https://github.com/projectatomic/adb-utils/issues/154

**Comment by hferentschik** [comment]
@budhrg Is this still an issue? The line you are referencing in the issue seems out of date.

**Comment by brgnepal** [comment]
@hferentschik These are the lines I am referring https://github.com/projectatomic/vagrant-service-manager/blob/master/features/support/env.rb#L110-L155

**Comment by hferentschik** [comment]
As reference:

```
# Note: Only for services supported through systemctl. Not for 'kubernetes'.
Then(/^the service "([^"]*)" should be ([^"]*)$/) do |service, operation|
  run("vagrant ssh -c \"sudo systemctl status #{service}\"")

  if %w(running restarted).include? operation
    exit_code = 0
    regexp = /Active: active \(running\)/
  elsif operation == 'stopped'
    exit_code = 3
    regexp = /Active: inactive\(dead\)/
  end

  expect(last_command_started).to have_exit_status(exit_code)
  aruba.command_monitor.find(Aruba.platform.detect_ruby(last_command_started)).send(:stdout) =~ regexp
end
```

Got you now. Why do you think using sccli would be better in this case? After all it is also just wrapping around _systemd_, right?

**Comment by brgnepal** [comment]
@hferentschik , I thought that since `sscli` does support all the service operations but I found later that the only way we can validate is using exit code that `sccli` return.

The other possible argument of using `sccli` is, this way we are actually testing `sccli` properly. 
Another issue is we cannot test kubernetes using this step definition since there is nothing like `systemctl status kubernetes`.

Do you think this line

```
expect(last_command_started).to have_exit_status(exit_code)
```

is enough in the step definition to validate service is running or restarted?

**Comment by brgnepal** [comment]
Closing this now since there is not enough way to validate service is running or restarted. In case of `systemd` it provides logs on `status` command which we can use to verify service status.
